lib/qb_backend/posts/manual.ex

```diff
@@ -53,4 +59,24 @@ defmodule QbBackend.Posts.Manual do
     |> Manual.changeset(attrs)
     |> put_assoc(:profile, profile)
   end
+@doc """
+  This function takes a bookmark and a manual and creates an association between them
+  """
+  @spec add_to_bookmark(Manual.t(), Bookmark.t()) :: {:ok, Manual.t()} | {:error, Ecto.Changeset.t()}
+  def add_to_bookmark(%Manual{} = manual, %Bookmark{} = bookmark) do
+    manual
+    |> changeset(%{})
+    |> put_assoc(:bookmark, bookmark)
+  end
+
+  @doc """
+  This function  takes an item and removes the cart association from it
+  """
+  @spec remove_manual_from_bookmark(Manual.t()) :: {:ok, Manual.t()} | {:error, Ecto.Changeset.t()}
+  def remove_manual_from_bookmark(%Manual{bookmarks: _bookmark} = manual) do
+    manual
+    |> changeset(%{})
+    |> put_change(:bookmark_id, nil)
+    |> put_change(:bookmark, nil)
```

Comment:
This line is not relevant, we are going to need to find another way of building this associations as this strategy doesn't work for many to many associations

Reply:
erm can you check how the associations work on spender between an Item and a Logsection

Reply:
@zacck @br4inii I think we don't need the `add manual to bookmark` on the `manual.ex` since it's already captured on the `bookmark.ex` file. Plus the `manual.ex` file can already create a manual for us... What are your thoughts?

Reply:
@WanjikuMac  however way works for you and is most efficient ... remember `deleting code is better than adding`